packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -265,6 +271,84 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
     FlutterErrorDetails.propertiesTransformers.add(transformDebugCreator);
   }
 
+  void _debugAddStackFilters() {
+    const PartialStackFrame elementInflateWidget = PartialStackFrame(className: 'Element', method: 'inflateWidget', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame elementUpdateChild = PartialStackFrame(className: 'Element', method: 'updateChild', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame elementRebuild = PartialStackFrame(className: 'Element', method: 'rebuild', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame componentElementPerformRebuild = PartialStackFrame(className: 'ComponentElement', method: 'performRebuild', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame componentElementFristBuild = PartialStackFrame(className: 'ComponentElement', method: '_firstBuild', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame componentElementMount = PartialStackFrame(className: 'ComponentElement', method: 'mount', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame statefulElementFristBuild = PartialStackFrame(className: 'StatefulElement', method: '_firstBuild', packagePath: 'src/widgets/framework.dart');
+    const PartialStackFrame singleChildMount = PartialStackFrame(className: 'SingleChildRenderObjectElement', method: 'mount', packagePath: 'src/widgets/framework.dart');
```

Comment:
nit: it would be nice if these filters could include package:flutter in the package path to eliminate the small chance of false positives.

Reply:
Done